app/gooeyInput.tsx

```diff
@@ -17,9 +18,14 @@ export function GooeyTextarea({
   return (
     <div className="gui-input gui-input-textarea">
       {label && (
-        <label>
-          <RenderedMarkdown body={label} />
-        </label>
+        <span className="d-inline-block position-relative">
+          <label>
+            <RenderedMarkdown body={label} />
+          </label>
+          {help && (
+            <HelpTooltip content={help as string} placement={tooltip_placement} />
+          )}
+        </span>
```

Comment:
Do you wanna create a component for this? Given its used many times

Reply:
I don't think that's needed here, we might have to add extra classes at some places and not need label tag so it will be just a  couple of div with classes @devxpy

Reply:
@devxpy Made the changes